src/Utils.js

```diff
@@ -116,7 +116,26 @@ export function selectorError(selector) {
   );
 }
 
-export const isCompoundSelector = /[a-z]\.[a-z]/i;
+export const isCompoundSelector = /([a-z]\.[a-z]|[a-z]\[.*\])/i;
```

Comment:
Maybe test again? I literally just tested both those ideas and they both matched.